ingress/single-cluster/ingress-asm-multi-backendconfig/README.md

```diff
@@ -0,0 +1,152 @@
+# Anthos Service Mesh Ingress with multiple backend configs
+
+This recipe shows how to use a single ASM ingress gateway to run multiple different backends with different Backend Configs.
+This can be used to e.g. protect some content behind the IAP (Like Montoring Tools or Admin GUI).
+
+In order to use Backend Configs Istio Ingress Gateway needs to be exposed as a Layer7 (HTTP/HTTPS) Load Balancer, not as Layer 4 TCP Load balancer as it is by default.
+This means other TCP based protocols can't be exposed via the same Ingress/External IP Address.
+
```

Comment:
One could also go into the other advantages of this approach, apart from being able to "only" use multiple BackendConfigs with an IG. For example:

```suggestion
Replacing the default Kubernetes `LoadBalancer` service with an Ingress resource to get traffic to an Istio Ingress Gateway generally allows you to combine the features of Istio Ingress Gateways resource with the features of a [External HTTP(S) load balancer](https://cloud.google.com/load-balancing/docs/https) and [GKE ingress](https://cloud.google.com/kubernetes-engine/docs/how-to/ingress-features): besides Backend Configs, this also allows for example to use global load balancing or FrontendConfigs (such as for HTTP-to-HTTPS redirect).

```